docs/release-notes/8.7.asciidoc

```diff
@@ -5,6 +5,94 @@
 [[release-notes-8.7.1]]
 === 8.7.1
 
+[discrete]
+[[known-issue-8.7.1]]
+==== Known issues
+
+* If you modify an exception item using the <<exceptions-api-update-item,update exception item>> API and _only_ specify its `item_id`, the exception item is erroneously duplicated. To avoid this issue, you can:
+
+** <<manage-exception,Update exception items>> through the {security-app} UI. 
+** Specify an exception item's `item_id` _and_ its `id` when modifying an exception through the <<exceptions-api-update-item,update exception item>> API. 
+** Only specify an exception item's `id` when updating an exception through the update exception item API. 
```

Comment:
It's not clear (at least to me) that this list is ORed together. Suggestion:

```suggestion
* If you modify an exception item using the <<exceptions-api-update-item,update exception item>> API and _only_ specify its `item_id`, the exception item is erroneously duplicated. To avoid this issue, you can either:

** <<manage-exception,Update exception items>> through the {security-app} UI
** Specify an exception item's `item_id` _and_ its `id` when modifying an exception through the <<exceptions-api-update-item,update exception item>> API. 
```